.github/merge_rules.yaml

```
@@ -236,6 +236,21 @@
   - Lint
   - pull
 
+- name: XPU ATen
+  patterns:
+  - third_party/xpu.txt
```

Don't you want to add more `c10/xpu`
```suggestion
  - aten/src/ATen/xpu/**
  - c10/xpu/**
  - third_party/xpu.txt
```